Created test examples for automationpractice.com

kmcadams opened the pull request:
Added new tests that uses automationpractice.com.  Also began adding cucumber tests and defined step definitions for the same site

Commit:
Changed to assertj for fluent assertions

Commit:
Changed to use 5.0.5-SNAPSHOT of atf-toolbox

kmcadams committed:
added test-output, bin, and settings to .gitignore

kmcadams committed:
changed base-url to automationpractice.com

kmcadams committed:
changed to use MacOS firefox 45.0

kmcadams committed:
changed atf-toolbox.version to 5.0.6-WIP

kmcadams committed:
Created practiceautomation LoadableComponent and PageObject classes

kmcadams committed:
Changed private method to public

kmcadams committed:
Added new webelement interactions

kmcadams committed:
Created Initial class for IndexPage test

kmcadams committed:
Created new feature file with two scenarios

kmcadams committed:
Added Cucumber dependency

kmcadams committed:
updated the toolbox snapshot version

kmcadams committed:
added more webelements and methods

kmcadams committed:
Added new test

kmcadams committed:
Added and implemented a prestepdefinitions class

kmcadams committed:
Created inital class for Contact Us page

srufle commented:
After I publish the new toolbox with Cucumber you will need to change your pom.xml to reference 5.1.x instead of 5.0.6-WIP

kmcadams committed:
changed toolbox version to use 5.1.2-SNAPSHOT

kmcadams committed:
changed the xml text formatting

kmcadams committed:
modified local version to use chrome by default 
and to pass 'ignore' as a default capability

kmcadams committed:
fixed the contactUs linktext webelement and the ...isDisplayed() method

kmcadams committed:
finished writing the webelements and corresponding interaction methods

kmcadams committed:
Fixed bad import

kmcadams committed:
Created contactUsPageTest class

Class has three test methods that assert the form is there 
and that email validation works

kmcadams committed:
Changed to use automationpractice class

kmcadams committed:
Added an AfterClass webdriver teardown method

srufle committed:
Merge branch 'master' into kyle-wip

srufle commented on candystore-automation/src/test/resources/sample-test-environment.xml:
```diff
@@ -4,13 +4,13 @@
-			<browser-download-path>C:\Program Files (x86)\Mozilla Firefox\31.0\firefox.exe</browser-download-path>
+			<browser-name>chrome</browser-name>
+<!-- 			<browser-version>45.0.2</browser-version> -->
+			<browser-download-path>/Users/kylemcadams/drivers/chromedriver</browser-download-path>
```
No kylemcadams in config files

kmcadams replied:
sorry, changing

kmcadams committed:
removed local file paths
